chrome/manifest.json

```diff
@@ -4,12 +4,10 @@
 	"manifest_version": 2,
 	"minimum_chrome_version": "26.0",
 	"description": "Reddit Enhancement Suite - a group of enhancements for browsing Reddit",
-	"background": {
-		"scripts": [
-			"vendor/polyfill.min.js",
-			"background.js"
-		]
-	},
+    "background": {
```

Comment:
Tab indentation pls. You can probably fix it from this page: click the pencil in the top right.